components/filter/events.tsx

```diff
@@ -170,6 +170,7 @@ const useEvents = (apiUrl: string) => {
       .then((json) => {
         setEvents(json);
       })
+      // eslint-disable-next-line no-console
       .catch((err) => console.error(err));
```

Comment:
I know you didn't add this code, but instead of `console.error` this error should be logged in App Insights

Reply:
Do we need to use trackException on every place where we use console.error?